openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -122,9 +121,14 @@ def horiz_comp_to_geom_mean(self, ctx, imt):
                     conv_sigma = (C[2] + (C[3]-C[2]) *
                                   np.log10(T/0.15)/np.log10(0.8/0.15))
                 rstd = C[4]
+    elif comp in ['GEOMETRIC_MEAN']:
+        conv_median = 1
+        conv_sigma = 1
```

Comment:
Is this correct? (also line 129). It means that the total_std is changed even if the median is not, and leads to invalid value in sqrt errors

Reply:
You're right. convert_sigma must be = 0